src/vm/arm/memcpy.S

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+//
+
+#include "unixasmmacros.inc"
+#include "asmconstants.h"
+
+.syntax unified
+.thumb
+
+//
+// void *memcpy(void *dst, const void *src, size_t length)
+//
+// Copy a block of memory in a forward direction.
+//
+
+    LEAF_ENTRY FCallMemcpy, _TEXT
+        b C_FUNC(memcpy)
```

Comment:
Please add the src / dst validation by attempting to access the memory they point to before jumping to the memcpy, like the JIT_MemCpy does.